src/Panel/panel.scss

```diff
@@ -54,42 +59,51 @@
         height: 100%;
         content: '';
         animation: loading 2s ease infinite;
-        background-color: $aquamarine;
+        background-color: $soft-loader-color;
       }
     }
   }
 
+  &.collapsable {
+    > .panel-header .panel-header-title-wrapper {
+      cursor: pointer;
+    }
+  }
+
   > .panel-header {
-    color: $white;
-    background-color: $charcoal;
+    color: $panel-header-color;
+    background-color: $panel-header-background-color;
+    padding: 0 12px;
 
     .expand-icon {
       font-size: 20px;
+      color: $expand-icon-color;
+      margin-right: 3px;
     }
 
     .panel-header-title {
       font-size: 16px;
-      font-weight: $semibold;
-      padding-left: 20px;
+      font-weight: $panel-header-title-font-weight;
```

Comment:
as discussed offline I personally wouldn't add a variable for `font-weight` but maybe in this case is useful